app/models/pageflow/entry.rb

```diff
@@ -43,6 +43,7 @@ class PasswordMissingError < StandardError
       create_draft!(home_button_enabled: theming.home_button_enabled_by_default)
       draft.storylines.create!(configuration: {main: true})
       theming.widgets.copy_all_to(draft)
+      theming.copy_defaults_to(draft)
```

Comment:
Actually `widgets.copy_all_to` is also about defaults. It sets up the default widgets configured in the theming for a new entry.